src/content/docs/extensions/full-text-search.md

```diff
@@ -0,0 +1,201 @@
+│ The Dragon's Call          │ 1.208044 │
+│ Chronicles of the Universe │ 0.380211 │
+└────────────────────────────┴──────────┘
+```
+
+### Drop FTS index
+
+Use the function `DROP_FTS_INDEX` to drop the FTS index on a table:
+
+```cypher
+CALL DROP_FTS_INDEX('TABLE_NAME', 'INDEX_NAME')
+```
+
+The example below shows how to drop the `book_index` index from the `Book` table:
+
+```cypher
+CALL DROP_FTS_INDEX('Book', 'book_index')
+```
+
+### Show FTS indexes
+
+There is no function specifically to show FTS indexes, but there is a general function [`SHOW_INDEXES`](/cypher/query-clauses/call) that
+can be used to show all the indexes available in the database.
+
+```cypher
+CALL SHOW_INDEXES() RETURN *;
+```
+This will return a list of all the indexes available in the database, while also listing the type of each
+index. Scan the table to find the FTS indexes that are currently available.
+
+```
+┌────────────┬─────────────┬────────────┬─────────────────────────┬──────────────────┬─────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ table name │ index nam   │ index type │ property names          │ extension loaded │ index definition                                                                                    │
+│ STRING     │ STRING      │ STRING     │ STRING[]                │ BOOL             │ STRING                                                                                              │
+├────────────┼─────────────┼────────────┼─────────────────────────┼──────────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────┤
+│ book       │ book_index  │ FTS        │ [abstract,author,title] │ True             │ CALL CREATE_FTS_INDEX('book', 'book_index', ['abstract', 'author', 'title' ], stemmer := 'porter'); │
+└────────────┴─────────────┴────────────┴─────────────────────────┴──────────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+### Prepared statement
+
+[Prepared statements](/get-started/prepared-statements) allows you to execute a query with different parameter values without rebinding the same query.
+A typical use case where parameters are useful is when you want to find books with different contents.
+
+Example:
+Let's start with preparing a cypher statement which queries the `book_index`.
+```c++
+auto preparedStatement = conn->prepare("CALL QUERY_FTS_INDEX('Book', 'book_index', $q) RETURN _node.ID, score;");
+```
+Now, we can find books with different contents using the prepared statement without rebinding.
+
+#### Find books related to `machine learning`
+```c++
+auto result = conn->execute(prepared.get, std::make_pair(std::string("q"), std::string("machine learning")));
+```
+
+#### Find books related to `dragons`
+```c++
+auto result = conn->execute(prepared.get, std::make_pair(std::string("q"), std::string("dragons")));
+```
```

Comment:
Is the `STEM` function supposed to be part of the public API? E.g., `RETURN STEM('knows', 'wrong')` from the tests. If yes, we should add that to the docs as well?

Reply:
I don't think users will be using the `STEM` function even if it's exposed publicly - is this something we expect anybody to use? It doesn't seem like a standard use of FTS.